<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachProcessDialog</class>
 <widget class="QWidget" name="AttachProcessDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label"/>
   </item>
   <item>
    <widget class="QLineEdit" name="processEdit"/>
   </item>
   <item>
    <widget class="QListView" name="processList"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
